src/test/java/org/galatea/starter/entrypoint/IexRestControllerTest.java

```diff
@@ -82,4 +83,20 @@ public void testGetLastTradedPriceEmpty() throws Exception {
         .andExpect(jsonPath("$", is(Collections.emptyList())))
         .andReturn();
   }
+
+  @Test
+  public void testGetHistoricalPrice() throws Exception {
```

Comment:
Add tests for exception scenarios: i.e., invalid request, IEX server failure

Reply:
@CurtisScheffel It looks like the `IEX server failure` scenario still might not be present in your tests. To test this, you could make a new wiremock mapping where someone calling `/stock/FAIL/chart/1m?token=sampleToken` will return a 500 error from the server. In this case, your FUSE program test would probably also return a 500 error `status().is5xxServerError()`